makefile

```diff
@@ -0,0 +1,9 @@
+install:
+	npm install
+
+run:
+	nodemon server.js
+
+test:
+	@./node_modules/.bin/mocha
```

Comment:
Remove `@`, you have `make --silent` for that. Furthermore you can add the `node_modules/.bin` to the PATH in the makefile to make it more readable. Check others projects.